test/README.testsuite

```diff
@@ -35,12 +35,12 @@ $ meson configure build -Dintegration-tests=true -Dremote=enabled -Dopenssl=enab
 
 Once enabled the integration tests can be run with:
 
-$ sudo meson test -C build/ --suite integration-tests --num-processes "$((nproc / 2))"
+$ meson test -C build/ --suite integration-tests --num-processes "$((nproc / 2))"
```

Comment:
Typo when I changed away from using getconf. Meson 1.0.1 will happily accept `--num-processes 0` and then deadlock.

```suggestion
$ meson test -C build/ --suite integration-tests --num-processes "$(($(nproc) / 2))"
```